src/main/scala/package.scala

```diff
@@ -30,9 +30,9 @@ package object constellation extends POWExt
   with SignHelpExt
   with KeySerializeJSON {
 
-  implicit var StandardTimeout: Timeout = Timeout(5, TimeUnit.SECONDS)
+  implicit var standardTimeout: Timeout = Timeout(5, TimeUnit.SECONDS)
 
-  val minimumTime : Long = 1518898908367L
+  final val MinimumTime : Long = 1518898908367L
```

Comment:
this isn't an objection -- but what is this time?

Reply:
Should be changed regularly. Just a convention in the other crypto repos of having a minimum known time for when the main net started